pysiaf/source_data/NIRCam/nircam_siaf_aperture_definition.txt

```diff
@@ -49,6 +49,33 @@
 #            NRCB5_SUB400P  ## updated to match B1_400P   
 #              NRCB5_FULLP  ## updated to match B1_FULLP
 #
+#
+# gennaro 2019-12-12
+#  Added 96 WFSS filter-dependent SLIT type apertures for visualization. Apertures for both LW detectors, and for
```

Comment:
Remove JWSTSIAF-67 references from this PR

Reply:
Addressed in incoming commit